torch/csrc/dynamo/eval_frame.c

```diff
@@ -646,6 +646,31 @@ static PyObject* _custom_eval_frame(
       frame->f_lasti,
       frame->f_iblock,
       frame->f_executing);
+
+  if (throw_flag) {
+    // When unwinding generators, eval frame is called with throw_flag ==
+    // true.  Frame evaluation is supposed to continue unwinding by propagating
+    // the exception.  Dynamo doesn't really know how to do this, nor does it
+    // really want to do this, because there's literally no code to
```

Comment:
"there's literally no code to capture" -> "it's unlikely there is any relevant code to capture"
To better match reality and the NB bellow that does mention __exit__ handlers.